### DISCUSSION TITLE
LINE Bot- 照片讀取不同步

### DISCUSSION THREAD

**Discussion by wolfyeva** [1101-GS3073-C/Python-Community-GS3073-C#343] [Q&A]
# 問題
我目前的功能為回傳使用者傳的照片，在手機上可以回傳正確的照片，但在電腦版的LINE上卻是app.py執行後使用者第一張回傳的照片，而在/static/images 裡是顯示和手機一樣正確的照片。

請問是電腦版LINE的Bug，還是我的程式碼有問題？
## 程式碼如下:
```
message_content = line_bot_api.get_message_content(event.message.id)
file = bytearray()
for chunk in message_content.iter_content():
    file.extend(chunk)
    stream = io.BytesIO(file)
    with open('static/images/temp.jpg', 'wb') as fd:
        fd.write(stream.read())
    line_bot_api.reply_message(event.reply_token,
        ImageSendMessage(
            original_content_url='https://f3fe-140-115-225-75.ngrok.io/static/images/temp.jpg',
            preview_image_url='https://f3fe-140-115-225-75.ngrok.io/static/images/temp.jpg'))
```

## 手機(正確)
![195320](https://user-images.githubusercontent.com/81672797/171544430-b1e9f9d2-c4c9-4582-a2ec-df0d5e5f3b3f.jpg)

## 電腦版(錯誤)
![螢幕擷取畫面 2022-06-02 110428](https://user-images.githubusercontent.com/81672797/171544512-dc348893-5b75-4283-806a-38ad6b482355.png)

## 網站(正確)
![螢幕擷取畫面 2022-06-02 110412](https://user-images.githubusercontent.com/81672797/171544547-88529bd6-201b-4f48-813c-6124af99b440.png)

**Comment by Bill671** [comment]
哈囉，方便請教一下後來有找到解決辦法嗎
